Assets/Scripts/FinalBoss/FBGameManager.cs

```diff
@@ -159,14 +140,21 @@ public void GoToMainMenu()
         LevelChanger.instance.FadeToLevel((int)Constants.gameScenes.MAINMENU);
     }
 
-    public void RemoveEnemy()
-    {
-        minionsOnScreen -= 1;
-    }
-
     public void RetrunMainMenu()
     {
         LevelChanger.instance.FadeToLevel((int)Constants.gameScenes.MAINMENU);
         Time.timeScale = 1f;
     }
+
+    public bool CheckMinions()
+    {
+        for (int i = 0; i< minionParent.transform.childCount; i++)
```

Comment:
for future notes, a foreach loop here on children might have been slightly more efficient.